### PR TITLE
Update js signature library to JSignature

### PR DESCRIPTION
**Pull request by levkar**
This PR updates js signature library to JSignature since it works with mobile as well.

This is the same update original author made in their own repo. Here's the related commit:

https://github.com/JayVora-SerpentCS/SerpentCS_Contributions-v8/commit/2d6c09482d9c553f8d8469ae869bbe5fc265fc93

This can be considered as a sync.

### CODE REVIEW

**Comment by eLBati** [comment]
@JayVora-SerpentCS @archetipo what do you think about this?

**Comment by JayVora-SerpentCS** [comment]
Sounds right, how about you @Meet_scs

**Comment by ghost** [comment]
@eLBati @JayVora-SerpentCS looks good. :+1:

File: web_widget_digitized_signature/static/src/js/digital_sign.js [hunk 1]
```diff
@@ -7,23 +7,42 @@ openerp.web_widget_digitized_signature = function(instance) {
...
+        initialize_content: function() {
+            this._super();
+            this.$el.find("#signature").empty().jSignature({'decor-color' : '#D1D0CE', 'color': '#000', 'background-color': '#fff'});
+            this.$el.find("#signature").attr({"tabindex": "0",'height':"100"});
+            this.empty_sign = this.$el.find("#signature").jSignature("getData",'image');
+            this.$el.find('#sign_clean').click(this.on_clear_sign);
+            this.$el.find('.save_sign').click(this.on_save_sign);
+        },
+        on_clear_sign: function() {
+            if (this.get('value') !== false) {
```
Comment (ghost) [inline web_widget_digitized_signature/static/src/js/digital_sign.js:20]: I suggest you to directly check for the value.

if(this.get('value') && !_.isUndefined(value)){
// your code here.
}

**Pull request closed without merging**